apps/mobile/mobile-app/components/impact/impact-feature-card.tsx

```
@@ -0,0 +1,156 @@
+}
+
+// Helper function to convert Tailwind background classes to actual colors
+const getBackgroundColor = (bgClass: string): string => {
+	switch (bgClass) {
+		case 'bg-green-100':
+			return '#dcfce7'
+		case 'bg-blue-100':
+			return '#dbeafe'
+		case 'bg-orange-100':
+			return '#fed7aa'
+		case 'bg-white':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)